firestore/integration_test.go

```diff
@@ -1743,6 +1743,72 @@ func TestIntegration_FieldTransforms_Set(t *testing.T) {
 
 type imap map[string]interface{}
 
+func TestIntegration_Serialize_Deserialize_WatchQuery(t *testing.T) {
+	shouldBeFoundID := collectionIDs.New()
+
+	// Create documents for partiontioned query result
+	ctx := context.Background()
+	h := testHelper{t}
+	client := integrationClient(t)
+	cr1 := client.Collection(shouldBeFoundID)
+	dr1 := cr1.Doc("should-be-found-1")
+	h.mustCreate(dr1, map[string]string{"some-key": "should-be-found"})
+	defer h.mustDelete(dr1)
```

Comment:
Would it be a good idea to pull the above three lines of code into a function and return the `mustGet` function? It might shorten the test a little bit.

Reply:
Refactored the code